pkg/k8scontext/k8scontext_test.go

```diff
@@ -66,6 +66,18 @@ var _ = Describe("K8scontext", func() {
 		},
 	}
 
+	// Create the Ingress resource.
+	pod := &v1.Pod{
```

Comment:
It sounds like we create similarly shaped objects in a few places within this file.  Perhaps it would be helpful to add a function that makes these (given a few variable parameters).